lib/src/migrators/module/references.dart

```diff
@@ -194,16 +211,57 @@ class _ReferenceVisitor extends RecursiveAstVisitor {
     _importer = importer;
     _scope = Scope();
     _moduleScopes[stylesheet.span.sourceUrl] = _scope;
+    _declarationSources = {};
+    _moduleSources[stylesheet.span.sourceUrl] = _declarationSources;
     visitStylesheet(stylesheet);
     _checkUnresolvedReferences(_scope);
+    _resolveBuiltInFunctionReferences();
     return References._(
         _variables,
         _variableReassignments,
         _defaultVariableDeclarations,
         _mixins,
         _functions,
         _getFunctionReferences,
-        _globalDeclarations);
+        _globalDeclarations,
+        _referenceSources);
+  }
+
+  void _resolveBuiltInFunctionReferences() {
+    var functions = _unresolvedReferences.keys.whereType<FunctionExpression>();
+    for (var function in functions) {
+      if (_isCssCompatibilityOverload(function)) continue;
+      if (function.name.asPlain == null) continue;
+      var name = function.name.asPlain;
+      if (builtInFunctionModules.containsKey(name)) {
+        var module = builtInFunctionModules[name];
```

Comment:
Style nit: rather than calling `containsKey()` followed by `[]`, it's usually better to just call `[]` and check if it's `null` (basically always unless the map can have meaningful `null` values). It's less repetitive and requires fewer lookups.